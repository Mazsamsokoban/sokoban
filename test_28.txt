loadtest 28
showstate shole
move w1 right
showmaze