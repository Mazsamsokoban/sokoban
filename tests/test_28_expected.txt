shole:w2 f1:w1 f2:box sw:null 

name:shole
friction:Normal
thing:w2
open:false

shole:null f1:null f2:w1 sw:box 

